### PR TITLE
Merge 4.7.1

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| demand-backup | passed |
| data-migration-gcs | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |
| self-healing | failed |
| clone-cluster | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/110/commits/e815690ad5de60a1ed88c14fbf205de4092eaf83
image: `perconalab/percona-postgresql-operator:PR-110-e815690a-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-110-e815690a-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-110-e815690a-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-110-e815690a-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-110-e815690a-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-110-e815690a-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |
| self-healing | failed |
| demand-backup | passed |
| data-migration-gcs | passed |
| clone-cluster | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/110/commits/e815690ad5de60a1ed88c14fbf205de4092eaf83
image: `perconalab/percona-postgresql-operator:PR-110-e815690a-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-110-e815690a-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-110-e815690a-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-110-e815690a-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-110-e815690a-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-110-e815690a-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | failed |
| init-deploy | passed |
| data-migration-gcs | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |
| self-healing | passed |
| clone-cluster | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/110/commits/39e0c59d66964218062b518b1adc5fef83c9a98d
image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | passed |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |
| self-healing | passed |
| clone-cluster | passed |
| data-migration-gcs | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/110/commits/39e0c59d66964218062b518b1adc5fef83c9a98d
image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-postgresql-operator/pull/110/commits/39e0c59d66964218062b518b1adc5fef83c9a98d
image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | passed |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |
| self-healing | passed |
| clone-cluster | passed |
| data-migration-gcs | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/110/commits/39e0c59d66964218062b518b1adc5fef83c9a98d
image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-110-39e0c59d-pgo-deployer`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- PGO Image Tag: (e.g. `centos8-4.7.1`)
+- Postgres Version (e.g. `13`)
+- Storage: (e.g. `hostpath`, `nfs`, or the name of your storage class)
+
+## Steps to Reproduce
+
+### REPRO
+
+Provide steps to get to the error condition:
+
```
Comment (cap1984) [inline .github/ISSUE_TEMPLATE/bug_report.md:41]: Why do we need this file?

File: deploy/operator.yaml [hunk 2]
```diff
@@ -174,15 +178,15 @@ data:
...
-    pgo_client_version: "4.6.2"
+    pgo_client_version: "4.7.1"
     pgo_cluster_admin: "false"
     pgo_disable_eventing: "false"
     pgo_disable_tls: "false"
     pgo_image_prefix: "perconalab/percona-postgresql-operator"
     pgo_image_pull_policy: "Always"
     pgo_image_pull_secret: ""
     pgo_image_pull_secret_manifest: ""
-    pgo_image_tag: "main"
```
Comment (cap1984) [inline deploy/operator.yaml:185]: Keeps Percona values

File: docs/config.toml [hunk 3]
```diff
@@ -25,7 +25,11 @@ disableNavChevron = false # set true to hide next/prev chevron, default is false
 highlightClientSide = false # set true to use highlight.pack.js instead of the default hugo chroma highlighter
 menushortcutsnewtab = true # set true to open shortcuts links to a new tab/window
 enableGitInfo = true
+<<<<<<< HEAD
 operatorVersion = "4.7.0"
+=======
+operatorVersion = "4.7.1"
+>>>>>>> v4.7.1
```
Comment (cap1984) [inline docs/config.toml:32]: Merge artifacts

File: docs/content/Configuration/compatibility.md [hunk 4]
```diff
@@ -12,7 +12,29 @@ version dependencies between the two projects. Below are the operator releases a
 
 | Operator Release   |      Container Release      | Postgres | PgBackrest Version
 |:----------|:-------------|:------------|:--------------
+<<<<<<< HEAD
 | 0.1.0 | 0.1.0  | 13.2 | 2.31 |
+=======
```
Comment (cap1984) [inline docs/content/Configuration/compatibility.md:17]: Merge artifacts

File: docs/content/custom-resources/_index.md [hunk 5]
```diff
@@ -345,7 +345,11 @@ The below code will help you set up this Secret.
 pgo_cluster_name=hippo
 # this variable is the namespace the cluster is being deployed into
 cluster_namespace=pgo
+<<<<<<< HEAD
```
Comment (cap1984) [inline docs/content/custom-resources/_index.md:348]: Merge artifacts

File: docs/content/releases/4.7.0.md [hunk 6]
```diff
@@ -135,7 +135,11 @@ Note that PGO will rewrite some of your HBA rules when performing any TLS enable
 # Features
 
 - Custom labels can now be added and removed through the `userlabels` attribute on the `pgclusters.crunchydata.com` custom resource, in addition to extending this functionality of the `pgo label` and `pgo delete label` commands.
+<<<<<<< HEAD
 - Custom labels are now applied to all of the managed Kubenretes objects associated with a Postgres cluster. These include: Deployments, Jobs, Pods, PVCs, ConfigMaps, and Secrets.
+=======
```
Comment (cap1984) [inline docs/content/releases/4.7.0.md:140]: Merge artifacts

File: installers/olm/Makefile [hunk 7]
```diff
@@ -11,7 +11,11 @@ OLM_TOOLS ?= registry.localhost:5000/postgres-operator-olm-tools:$(OLM_SDK_VERSI
 OLM_VERSION ?= 0.15.1
 PGO_BASEOS ?= centos8
 PGO_IMAGE_PREFIX ?= registry.developers.crunchydata.com/crunchydata
+<<<<<<< HEAD
```
Comment (cap1984) [inline installers/olm/Makefile:14]: Merge artifacts

File: pkg/apis/crunchydata.com/v1/doc.go [hunk 8]
```diff
@@ -82,15 +82,19 @@ $PGO_APISERVER_URL/showclusters
 curl --cacert $PGO_CA_CERT --key $PGO_CLIENT_KEY --cert $PGO_CA_CERT -u \
 admin:examplepassword -H "Content-Type:application/json" --insecure -X \
 POST --data \
+<<<<<<< HEAD
```
Comment (cap1984) [inline pkg/apis/crunchydata.com/v1/doc.go:85]: Merge artifacts

File: .github/ISSUE_TEMPLATE/support---question-and-answer.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+about: "Learn how to interact with the PGO community"
+---
+
+If you believe you have found have found a bug, please open up [Bug Report](https://github.com/CrunchyData/postgres-operator/issues/new?template=bug_report.md)
+
+If you have a feature request, please open up a [Feature Request](https://github.com/CrunchyData/postgres-operator/issues/new?template=feature_request.md)
+
+You can find information about general PGO [support](https://access.crunchydata.com/documentation/postgres-operator/latest/support/) at:
+
+[https://access.crunchydata.com/documentation/postgres-operator/latest/support/](https://access.crunchydata.com/documentation/postgres-operator/latest/support/)
```
Comment (qJkee) [inline .github/ISSUE_TEMPLATE/support---question-and-answer.md:12]: we need to rework this file.
Like remove crunchy links, change github issues links to our jira
@spron-in

Comment (nonemax) [reply .github/ISSUE_TEMPLATE/support---question-and-answer.md:12]: we should remove issue_template at-all

**Review by qJkee — APPROVED** [review]

File: installers/olm/Makefile (hunk 7 above)
Comment (nonemax) [reply installers/olm/Makefile]: https://github.com/percona/percona-postgresql-operator/pull/110/commits/33aef806e0d7e3573f887a278eb01127e0089d81

File: docs/content/releases/4.7.0.md (hunk 6 above)
Comment (nonemax) [reply docs/content/releases/4.7.0.md]: https://github.com/percona/percona-postgresql-operator/pull/110/commits/33aef806e0d7e3573f887a278eb01127e0089d81

File: docs/content/custom-resources/_index.md (hunk 5 above)
Comment (nonemax) [reply docs/content/custom-resources/_index.md]: https://github.com/percona/percona-postgresql-operator/pull/110/commits/33aef806e0d7e3573f887a278eb01127e0089d81

File: docs/content/Configuration/compatibility.md (hunk 4 above)
Comment (nonemax) [reply docs/content/Configuration/compatibility.md]: https://github.com/percona/percona-postgresql-operator/pull/110/commits/33aef806e0d7e3573f887a278eb01127e0089d81

File: docs/config.toml (hunk 3 above)
Comment (nonemax) [reply docs/config.toml]: https://github.com/percona/percona-postgresql-operator/pull/110/commits/33aef806e0d7e3573f887a278eb01127e0089d81

File: deploy/operator.yaml (hunk 2 above)
Comment (nonemax) [reply deploy/operator.yaml]: https://github.com/percona/percona-postgresql-operator/pull/110/commits/33aef806e0d7e3573f887a278eb01127e0089d81

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (nonemax) [reply .github/ISSUE_TEMPLATE/bug_report.md]: https://github.com/percona/percona-postgresql-operator/pull/110/commits/33aef806e0d7e3573f887a278eb01127e0089d81

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | passed |
| init-deploy | passed |
| data-migration-gcs | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |
| self-healing | passed |
| clone-cluster | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/110/commits/33aef806e0d7e3573f887a278eb01127e0089d81
image: `perconalab/percona-postgresql-operator:PR-110-33aef806-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-110-33aef806-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-110-33aef806-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-110-33aef806-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-110-33aef806-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-110-33aef806-pgo-deployer`

**Pull request merged**